NEWS.md

```diff
@@ -3,6 +3,237 @@ XGBoost Change Log
+language binding you are using for details. Lastly, default internal serialization format
+is set to UBJSON, which affects Python pickle and R RDS. (#7572, #7570, #7358, #7571,
+#7556, #7549, #7416)
+
+### General new features
+Other than the major new features mentioned above, some others are summarized here:
+
+* Users can now access the build information of XGBoost binary in Python and C
+  interface. (#7399, #7553)
+* Remove auto configuration of `seed_per_iteration`, now distributed training should
+  generate closer result to single node training when sampling is used. (#7009)
+* A new parameter `huber_slope` is introduced for the `Pseudo-Huber` objective.
+* During source build, XGBoost can choose cub in system path automatically. (#7579)
+* XGBoost now honors the CPU counts from CFS, which is usually set in docker
+  environment. (#7654, #7704)
+* The metric `aucpr` is rewritten for better performance and GPU support. (#7297, #7368)
+* Metric calculation is now performed in double precision. (#7364)
+* XGBoost no longer mutate the global OpenMP thread limit. (#7537, #7519, #7608, #7590, #7589, #7588)
+* The default behavior of `max_leave` and `max_depth` is now unified (#7302, #7551).
+* CUDA fat binary is now compressed. (#7601)
+* Use double for GPU Hist node sum, which improves the accuracy of `gpu_hist`. (#7507)
+
+### Deterministic result for evaluation metric and linear model
+In previous versions of XGBoost, evaluation result might differ slightly for each run due
+to parallel reduction for floating point values, which is now addressed. (#7362, #7303,
+#7316, #7349)
+
+### Performance improvements
+Most of the performance improvements are integrated into other refactors during feature
+developments. The `approx` should see significant performance gain for many datasets as
+mentioned in previous section, while the `hist` tree method also enjoys improved
+performance with the removal of the internal `pruner` along with some other
+refactoring. Lastly, `gpu_hist` no longer synchronize the device during training. (#7737)
+
+### General bug fixes
+* Fixes in CMake script for exporting configuration. (#7730)
+* XGBoost can now handle unsorted sparse input. This includes text file format like libsvm
+  and scipy sparse matrix where column index is not sorted. (#7731)
+* Fix tree param feature type, this affects inputs with number of columns greater than the
+maximum value of int32. (#7565)
+* Fix external memory with gpu_hist and subsampling. (#7481)
+* Check number of trees in inplace predict, this avoids a potential segfault when an
+  incorrect value for `iteration_range` is provided. (#7409)
+
+### Changes in the Python package
+Other than the changes in Dask, the XGBoost Python package gained some new features and
+improvements along with small bug fixes.
+
+* Python 3.7 is required as the lowest Python version. (#7682)
+* Binary package Support Apple Silicon. (#7621, #7612)
```

Comment:
```suggestion
* Pre-built binary wheel for Apple Silicon. (#7621, #7612) Apple Silicon users will now be able to run `pip install xgboost` to install XGBoost.
```

Can you also merge in https://github.com/dmlc/xgboost/pull/7747 as part of 1.6 release?